FIND FACTORIAL OF A NUMBER

```diff
@@ -1,19 +1,13 @@
 # Python program to find the factorial of a number provided by the user.
 
-# change the value for a different result
-num = 7
+def factorial(n):
+   if n < 0:
+      return("Oops!Factorial Not Possible")
+   elif n = 0:
+      return 1
+   else:
+      return n*factorial(n-1)
 
-# To take input from the user
-#num = int(input("Enter a number: "))
+n = int(input())
```

Comment:
`n = int(input("Enter a number: "))`
This will be more good format to take input from the user by printing a message